pom.xml

```diff
@@ -60,12 +60,12 @@
 		<maven.compiler.target>11</maven.compiler.target>
 		<maven.compiler.source>11</maven.compiler.source>
 
-		<wicket.version>9.7.0</wicket.version>
-		<jackson.version>2.12.5</jackson.version>
-		<slf4j.version>1.7.25</slf4j.version>
-		<junit.version>5.8.2</junit.version>
+		<wicket.version>9.17.0</wicket.version>
+		<jackson.version>2.17.1</jackson.version>
```

Comment:
I'd recommend using the same versions Wicket uses, i.e. 2.14.3 (https://github.com/apache/wicket/blob/bc7dcc3f5bfbb3a41ab407d53ba5e0af3d3453a0/pom.xml#L159)